plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergV2.java

```diff
@@ -285,6 +287,98 @@ public void testSelectivelyOptimizingLeavesEqualityDeletes()
         Assertions.assertThat(loadTable(tableName).currentSnapshot().summary().get("total-equality-deletes")).isEqualTo("1");
     }
 
+    @Test
```

Comment:
Please add tests for handling equality deletes in the context of schema evolution (add/drop column from the table - omit dropping a column referenced from an equality delete file because this doesn't work at the moment)